ghost/core/core/server/services/email-service/email-provider-factory.js

```
@@ -0,0 +1,63 @@
+ * Future providers (ses, sendgrid, postmark) will be added in upcoming releases.
+ * DO NOT change this setting unless you're developing/testing new providers.
+ *
+ * @param {Object} config - Configuration service
+ * @returns {string} The provider name
+ * @throws {Error} If provider is not 'mailgun'
+ */
+function resolveEmailProvider(config) {
+    const provider = config.get('bulkEmail:provider') || 'mailgun';
+
+    if (provider !== 'mailgun') {
+        throw new Error(`Unknown bulk email provider: ${provider}. Only 'mailgun' is currently supported.`);
```

ESLint

[error] Direct calls to new Error() are not allowed. Please use @tryghost/errors.

(ghost/ghost-custom/no-native-error)